server.js

```diff
@@ -23,6 +55,61 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+// get(read) all the messages posted by the client into the database,
+// sort them by date in a descendent way and limit them to 20
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' })
+    .limit(20)
+    .exec()
+  res.json(thoughts)
+})
+
+// post(write) your new message into the database! 
+app.post('/thoughts', async (req, res) => {
+  // Retrieve the information sent by the client to our API endpoint
+  const { message } = req.body
+  // Use our mongoose model to create the database entry
+  const thought = new Thought({ message })
+  try {
+    // succesfull case, where all the data is correct and we can save it in our database
+    const savedThought = await thought.save()
+    res.status(201).json(savedThought)
+  } catch (err) {
+    res.status(400).json({ message: 'Message could not be saved', error: err.errors })
+  }
+})
+
+// When you want to update a message with likes (hearts): 
+// 1) Get the message that you want to update (by id)
+app.get('/thoughts/:id', (req, res) => {
+  Thought.findOne({ id: req.params._id }).then(thought => {
+    if (thought) {
+      res.json(thought)
+    } else {
+      res.status(404).json({ error: 'Not found', error: err.errors })
+    }
+  })
+})
```

Comment:
What are you doing with this endpoint? Do you need it? Are you calling this from the frontend at some point?